tests/services/test_path_reverser.py

```
@@ -171,15 +169,14 @@ def test_reverse_move_failure(self, db_session, tmp_path, monkeypatch):
         )
         db_session.add(record)
         db_session.commit()
-        
+
         def mock_move(src, dst):
```

Ruff (0.15.7)

[warning] Missing return type annotation for private function `mock_move`

Add return type annotation: `NoReturn`

(ANN202)